Cargo.toml

```
@@ -2,7 +2,7 @@
 version = "0.5.43"                                        # same as `hotshot`, but workspace subcrate can also release its own version
 authors = ["Espresso Systems <hello@espressosys.com>"]
 edition = "2021"
-rust-version = "1.65.0"
+rust-version = "1.76.0"
```

I agree, I think this should stay:

```suggestion
rust-version = "1.65.0"
```